youtube_dl_gui/locale/he-IL/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,588 @@
+
+#: youtube_dl_gui/optionsframe.py:481
+msgid "Extract audio from video file"
+msgstr "חלץ שמע מקבצי וידיאו"
+
+#: youtube_dl_gui/optionsframe.py:482
+msgid "Embed thumbnail in audio file"
+msgstr "הטמע תמונה ממוזערת בקובץ שמע"
+
+#: youtube_dl_gui/optionsframe.py:483
+msgid "Add metadata to file"
+msgstr "הוסף נתונים לקובץ"
+
+#: youtube_dl_gui/optionsframe.py:485
+msgid "Audio quality"
+msgstr "איכות השמע"
+
+#: youtube_dl_gui/optionsframe.py:543
+msgid "English"
+msgstr "אנגלית"
+
+#: youtube_dl_gui/optionsframe.py:544
+msgid "French"
+msgstr "צרפתית"
+
+#: youtube_dl_gui/optionsframe.py:545
+msgid "German"
+msgstr "גרמנית"
+
+#: youtube_dl_gui/optionsframe.py:546
+msgid "Greek"
+msgstr "יוונית"
+
+#: youtube_dl_gui/optionsframe.py:547
+msgid "Hebrew"
+msgstr "עברית"
+
+#: youtube_dl_gui/optionsframe.py:548
+msgid "Italian"
+msgstr "איטלקית"
+
+#: youtube_dl_gui/optionsframe.py:549
+msgid "Portuguese"
+msgstr "פורטוגזית"
+
+#: youtube_dl_gui/optionsframe.py:550
+msgid "Russian"
+msgstr "רוסית"
+
+#: youtube_dl_gui/optionsframe.py:551
+msgid "Spanish"
+msgstr "ספרדית"
+
+#: youtube_dl_gui/optionsframe.py:552
+msgid "Swedish"
+msgstr "שוודית"
+
+#: youtube_dl_gui/optionsframe.py:553
+msgid "Turkish"
+msgstr "טורקית"
+
+#: youtube_dl_gui/optionsframe.py:569
+msgid "None"
+msgstr "ריק"
+
+#: youtube_dl_gui/optionsframe.py:570
+msgid "Automatic subtitles (YOUTUBE ONLY)"
+msgstr "כתוביות אוטומטיות (YOUTUBE בלבד)"
+
+#: youtube_dl_gui/optionsframe.py:571
+msgid "All available subtitles"
+msgstr "כל הכתוביות הזמינות"
+
+#: youtube_dl_gui/optionsframe.py:572
+msgid "Subtitles by language"
+msgstr "כתוביות לפי שפה"
+
+#: youtube_dl_gui/optionsframe.py:578
+msgid "Subtitles"
+msgstr "כתוביות"
+
+#: youtube_dl_gui/optionsframe.py:582
+msgid "Subtitles options"
+msgstr "אפשרויות כתוביות"
+
+#: youtube_dl_gui/optionsframe.py:583
+msgid "Embed subtitles into video file (mp4 ONLY)"
+msgstr "תרגום כתוביות לקובץ וידאו (mp4 בלבד)"
+
+#: youtube_dl_gui/optionsframe.py:585
+msgid "Playlist"
+msgstr "רשימת השמעה"
+
+#: youtube_dl_gui/optionsframe.py:591 youtube_dl_gui/optionsframe.py:596
+msgid "Max"
+msgstr "מקסימלי"
+
+#: youtube_dl_gui/optionsframe.py:594
+msgid "Filesize"
+msgstr "גודל הקובץ"
+
+#: youtube_dl_gui/optionsframe.py:599
+msgid "Min"
+msgstr "מינימלי"
+
+#: youtube_dl_gui/optionsframe.py:728
+msgid "Retries"
+msgstr "נסיונות"
+
+#: youtube_dl_gui/optionsframe.py:731
+msgid "Authentication"
+msgstr "אימות"
+
+#: youtube_dl_gui/optionsframe.py:733
+msgid "Username"
+msgstr "שם משתמש"
+
+#: youtube_dl_gui/optionsframe.py:735
+msgid "Password"
+msgstr "סיסמא"
+
+#: youtube_dl_gui/optionsframe.py:737
+msgid "Video password"
+msgstr "סיסמת וידאו"
+
+#: youtube_dl_gui/optionsframe.py:740
+msgid "Network"
+msgstr "רשת"
+
+#: youtube_dl_gui/optionsframe.py:742
+msgid "Proxy"
+msgstr "פרוקסי"
+
+#: youtube_dl_gui/optionsframe.py:744
+msgid "User agent"
+msgstr ""
+
+#: youtube_dl_gui/optionsframe.py:746
+msgid "Referer"
+msgstr "מפנה"
```

Comment:
https://en.wikipedia.org/wiki/HTTP_referer
Your translation makes no sense

Reply:
https://he.wikipedia.org/wiki/Referer
As you can see in the first line on the hebrew wiki 
"ה-Referer (או דף **מפנה**) הוא כתובת ה-URL של דף האינטרנט הקודם שבו לחצו על קישור"